app/models/bookmark_search.rb

```diff
@@ -77,6 +79,73 @@ def process_options
     self.options.delete_if { |key, value| value.blank? }
   end
 
+  # Useful for inspecting the differences between the old ES query and the new
+  # ES query, when debugging problems between ES versions
+  def search_query
```

Comment:
I always meant to split this out, and I agree that it's useful, but even for temp code, can we avoid duplicating this? I think the only difference between this code and the query in search_results is the lack of pagination here - is there any way to add that back and then just call search_query within search_results?